server/app/views/applicant/NavigationFragment.html

```diff
@@ -25,7 +25,15 @@
               />
             </div>
             <em class="usa-logo__text">
-              <span th:text="${civicEntityShortName}"></span>
+              <span
+                class="cf-hidden-gov-name"
```

Comment:
suggestion (non-blocking): Would you add a comment here that briefly explains how this works? I was confused at first about why were including `civicEntityShortName` twice.